packages/api-client/src/v2/features/app/hooks/use-workspace-selector.ts

```diff
@@ -103,22 +103,22 @@ export const useWorkspaceSelector = (): UseWorkspaceSelectorReturn => {
 
     // For the dev mode we might want couple more documents for testing purpose
     // TODO: remove as soon as the command palette is implemented
-    if (import.meta.env.DEV) {
-      await draftStore.addDocument({
-        name: 'scalar-galaxy',
-        url: 'https://galaxy.scalar.com/openapi.yaml',
-      })
-
-      await draftStore.addDocument({
-        name: 'swagger-petstore-3-0',
-        url: 'https://petstore3.swagger.io/api/v3/openapi.json',
-      })
-
-      await draftStore.addDocument({
-        name: 'stripe',
-        url: 'https://raw.githubusercontent.com/stripe/openapi/refs/heads/master/openapi/spec3.json',
-      })
-    }
+    // if (import.meta.env.DEV) {
```

Comment:
@DemonHa do we still need these? we got persistence now can just load documents through the UI